### PR TITLE
Unbreak the tensorboard windows build

### LINKED ISSUE #1466: Vulcanizing failed while building Tensorboard
Hi,
I am getting the following error related to Vulcanizing while building Tensorboard:
**$ bazel run //tensorboard -- --logdir ./my_logs**
INFO: Analysed target //tensorboard:tensorboard (80 packages loaded).
INFO: Found 1 target...
**ERROR: C:/users/deepika.d.agarwal/desktop/what-if/tensorboard-master/tensorboard/components/BUILD:40:1: Vulcanizing /trace_viewer.html failed (Exit 1)**
Error: Could not find or load main class %1$tH:%1$tM:%1$tS.%1$tL
Caused by: java.lang.ClassNotFoundException: %1$tH:%1$tM:%1$tS.%1$tL
Target //tensorboard:tensorboard failed to build

Pls help if I am missing something here.
Thanks

### PR DESCRIPTION
**Pull request by nfelt**
This PR makes it possible to build a working TensorBoard on Windows again, which has evidently been broken for quite some time.  This in turn makes it possible for us to debug Windows-specific TensorBoard issues.

There are three separate fixes in this PR:
- Fixes to execrooter.py to work on Windows, e.g. by using copying instead of symlinks
- Removal of a bug-triggering `jvm_flags` argument to the vulcanize java_binary() rule - this is a bug in bazel's parsing of jvm_flags on windows, see https://github.com/bazelbuild/bazel/issues/7072, which for us manifested itself as #1466 
- Bumping our werkzeug dep to 0.11.15 to pick up fixes from https://github.com/pallets/werkzeug/pull/999 so it's compatible with Windows + python 3.6

With these fixes, I can build and run tensorboard at HEAD on Windows 10, using the hard-won magical incantation of:
- python 3.6.2
  - virtualenv 16.2.0
  - pip 18.1
  - tf-nightly 1.13.0.dev20190110 and its deps
- bazel version 0.21.0, plus its deps for installation plus building C++ and Java binaries (C++ is needed to build the protobuf runtime):
  - MSYS2 (msys2-x86_64-20180531.exe)
  - Visual Studio 2015 Community Edition with Visual C++
  - Windows 10 SDK
  - JDK 8 for windows, specifically build 1.8.0_191-b12
- git version 2.19.0.windows.1 (not part of the build itself, but part of getting this all working)

Fixes #1466

### CODE REVIEW

File: tensorboard/scripts/execrooter.py [hunk 1]
```diff
@@ -43,14 +43,23 @@ def run(inputs, program, outputs):
       parent = os.path.join(root, os.path.dirname(fake))
       if not os.path.exists(parent):
         os.makedirs(parent)
-      os.symlink(os.path.join(cwd, real), os.path.join(root, fake))
+      # Use symlink if possible and not on Windows, since on Windows 10
+	  # symlinks exist but they require administrator privileges to use.
```
Comment (wchargin) [inline tensorboard/scripts/execrooter.py]: nit: tabs messing up indentation

File: third_party/python.bzl [hunk 2]
```diff
@@ -69,11 +69,10 @@ def tensorboard_python_workspace():
     http_archive(
         name = "org_pocoo_werkzeug",
         urls = [
-            "https://mirror.bazel.build/pypi.python.org/packages/b7/7f/44d3cfe5a12ba002b253f6985a4477edfa66da53787a2a838a40f6415263/Werkzeug-0.11.10.tar.gz",
-            "https://pypi.python.org/packages/b7/7f/44d3cfe5a12ba002b253f6985a4477edfa66da53787a2a838a40f6415263/Werkzeug-0.11.10.tar.gz",
+			"https://files.pythonhosted.org/packages/fe/7f/6d70f765ce5484e07576313897793cb49333dd34e462488ee818d17244af/Werkzeug-0.11.15.tar.gz"
```
Comment (wchargin) [inline third_party/python.bzl]: nit: tabs messing up indentation

Comment (nfelt) [reply third_party/python.bzl]: Done.

File: tensorboard/scripts/execrooter.py (hunk 1 above)
Comment (nfelt) [reply tensorboard/scripts/execrooter.py]: Done.

**Pull request merged**